service/manager.go

```diff
@@ -265,8 +270,7 @@ func (m *manager) shutdown(err error, reason string, exitCode *int) (bool, error
 	}
 
 	m.transitionState(Stopped)
-
-	return true, err
+	return true, multierr.Combine(err, multierr.Combine(errs...))
```

Comment:
yarpc is using it: https://github.com/yarpc/yarpc-go/blob/dev/glide.yaml#L66
and they are on 1.0